### DISCUSSION TITLE
How to change Apache version?

### DISCUSSION THREAD

**Discussion by manuelmdn** [leokhoa/laragon#380] [Q&A] [answered]
Hello 

Unfortunately, I need to test an application using PHP 5.5, however, the Laragon Apache version is not compatible. Does anyone have an example on how to change the Apache version?

thanks in advance

![image](https://user-images.githubusercontent.com/1796188/215659763-60d56d5b-663c-4839-928d-aae5935588e3.png)

**Comment by Pen-y-Fan** [comment, marked as answer]
You can download a VC11 version from [Apache Lounge](https://www.apachelounge.com/download/additional/)

The process is similar to [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/). Except for the extracted files from the downloaded zip go into the **C:\laragon\bin\apache** directory.  You can switch versions from the Laragon > Apache > version [...] > menu.

**Comment by rescenic** [comment]
Your Apache VC version is too high. Use the same VC version of PHP and Apache.<br>
Apache:<br>
![https://i.postimg.cc/ZK9hPhjW/laragon-apache.png](https://i.postimg.cc/ZK9hPhjW/laragon-apache.png)
PHP:<br>
![https://i.postimg.cc/8Pqgtnnt/laragon-php.png](https://i.postimg.cc/8Pqgtnnt/laragon-php.png)
